### DISCUSSION TITLE
Setup Class C multicast - NucleoL073

### DISCUSSION THREAD

**Discussion by krispstack** [Lora-net/LoRaMac-node#1032] [Q&A] [answered]
Hi

How to Setup Class C multicast in periodic-uplink-lpp, Like in 1.0.2

https://github.com/Lora-net/LoRaMac-node/blob/v4.5.0/src/apps/LoRaMac/periodic-uplink-lpp/NucleoL073/main.c

Thanks

**Comment by ConnyBusy** [comment, marked as answer]
Maybe you want to have a look at

[https://github.com/Lora-net/LoRaMac-node/blob/v4.5.0/src/apps/LoRaMac/common/LmHandler/packages/LmhpRemoteMcastSetup.c](https://github.com/Lora-net/LoRaMac-node/blob/v4.5.0/src/apps/LoRaMac/common/LmHandler/packages/LmhpRemoteMcastSetup.c)

instead
